AppSyncRealTimeClient/Support/AppSyncURLHelper.swift

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright 2018-2021 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public enum AppSyncURLHelper {
+
+    // A standard app sync domain URL would look like
+    // https://abcdefghijklmnopqrstuvwxyz.appsync-api.us-west-2.amazonaws.com/graphql
```

Comment:
sorry, i'm still confused what is a standard appsync URL. how about correct this if i am wrong?
```
A standard AppSync URL has the format of 
`https://[DOMAIN].appsync-api.[REGION].amazonaws.com/graphql`
The regex `\w{26}` is used to check that the DOMAIN is [what the regex is doing].
The regex `\w{2}(?:(?:-\w{2,})+)-\d` is used to check that the REGION matches [what the regex is doing]. 
For example, "us-west-1'.
```